### ISSUE TITLE
Error impossible to open VRT file

### ISSUE THREAD

**Issue by avdbk** [camptocamp/QGIS-SpreadSheetLayers#54]
When I want to create a layer using the plugin, I get an error saying it's impossible to open VRT File <filename>.
I tried to uninstall and reinstall the plugin, but that didnt work. I tried different .xlsx files, but neither of them worked, they all gave the error.

Plugin version 2.1.2
QGIS version 3.28.4-Firenze

**Comment by arnaud-morvan** [comment]
Can you give the full error message and stack trace if available.

**Comment by avdbk** [comment]
![afbeelding](https://github.com/camptocamp/QGIS-SpreadSheetLayers/assets/168079756/def2d613-90d7-483f-8e5a-393cf10ad07b)

Impossible to open VRT file C:/Users/angela.vandenbroek/OneDrive - BIJ12/Documents/GIS/2024.05.30_HMNDR.xlsx.Blad1.vrt

**Comment by arnaud-morvan** [comment]
Nothing more in QGIS log messages panel ?
Does the file exists ?

**Comment by avdbk** [comment]
Yes the file does exist. I have this problem with all .xlsx files I tried so far. In the QGIS log messages panel it says the same error as above.

![afbeelding](https://github.com/camptocamp/QGIS-SpreadSheetLayers/assets/168079756/a11f0e86-162e-4c59-b844-89d9abb75952)
